### ISSUE TITLE
Why did you remove the Linux Mint theme ?

### ISSUE THREAD

**Issue by mb4e** [linuxmint/mint-themes#484]
Hello,

I'm curious about the decision to remove the "Linux Mint" theme. I used it on my Arch Linux installation and really liked it. So far, I haven't found any way to use that theme again.

Thank you for any information you can provide.

**Comment by clefebvre** [comment]
Hello,

It was removed simply because we maintained many other themes already.

Historically this one was the default theme for Linux Mint, the Cinnamon theme didn't require much maintenance since it was just there for technical reasons, and mint-x, mint-y didn't have their own Cinnamon themes. So we maintained one theme.

Then Mint-X got its own Cinnamon theme, so did Mint-Y, and its fork Mint-L... so that's three new themes to support. Mint switched to Mint-Y by default.

And now Cinnamon itself is highly maintained as something other distribution use.

It made no sense to continue to maintain a theme we no longer used/needed and which wasn't associated with the GTK themes, so it was removed.
